### PR TITLE
Automatically create an example OAuth application whenever importing the developer database.

### PR DESCRIPTION
**Pull request by jfly**
This makes things easier for people testing out TNoodle. (needed for https://github.com/thewca/tnoodle/pull/236)

### CODE REVIEW

**Comment by jonatanklosko** [comment]
I'd consider adding a single SQL query to the TNoodle README instead (a query creating the oauth app). I think it's not much effort and saves us from adding the code (if we do this for TNoodle, we will probably be ok with doing so for other apps and the file will grow for no reason).

That said, if you feel it's better to have the code, I'm ok with it =)

File: WcaOnRails/config/initializers/doorkeeper.rb [hunk 1]
```diff
@@ -89,8 +89,8 @@
   # Forces the usage of the HTTPS protocol in non-native redirect uris (enabled
   # by default in non-development environments). OAuth2 delegates security in
   # communication to the HTTPS protocol so it is wise to keep this enabled.
-  #
-  # force_ssl_in_redirect_uri !Rails.env.development?
+
+  force_ssl_in_redirect_uri Rails.env.production?
```
Comment (jfly) [inline WcaOnRails/config/initializers/doorkeeper.rb]: This is not directly related to my changes, but allowed me to clean up the oauth request spec a bit (previously it was using `save!(validate: false)` to create non https redirect_uris.
